src/decl.c

```diff
@@ -665,6 +665,18 @@ struct tnnt_achvmt_data tnnt_achievements[NUM_TNNT_ACHIEVEMENTS] = {
   {"What Have I Unleashed", "Make a demon lord escape the dungeon"},
   {"Walking Bullseye", "Have +30 AC"},
   {"Doorstopper", "Have a door resist 8 consecutive times when trying to open or close it"},
+  {"Look the Other Way", "Successfully bribe a mercenary"},
+  {"Practically Permanent", "Have more than 10,000 turns of a temporary intrinsic"},
+  {"Minesflayer", "Kill a mind flayer or master mind flayer in the Gnomish Mines"},
+  {"Hoist With Its Own Petard", "Have a monster kill itself with its own wand zap"},
+  {"Double Bones", "Encounter two bones piles on a single level"},
+  {"Mr. Bones' Wild Ride", "Find the bones of four former heroes in one game"},
```

Comment:
OK. I thought I remembered some people who misinterpreted achievements to mean "over the course of the tournament" and figured it would head off any potential for misunderstanding, but it may be that I made up those questions/complaints out of whole cloth: when I try to remember anything specific I can't.

I guess I should change existing achievements with similar language (e.g. Balanced Diet, Cat Lady) as well?

Reply:
Now that I think about it, these strings also go straight into the achievements db and get rendered on the site, so... it's fine.

Reply:
Yeah I guess I was thinking about it in that context more so than seeing them in-game.  I just force pushed again to remove the "rephrase achievements" commit.